CRM/Core/BAO/ActionSchedule.php

```diff
@@ -304,11 +304,11 @@ public static function sendMailings($mappingID, $now) {
             ->context('actionSearchResult', (object) $dao->toArray());
           foreach ($tokenProcessor->evaluate()->getRows() as $tokenRow) {
             if ($actionSchedule->mode == 'SMS' or $actionSchedule->mode == 'User_Preference') {
-              CRM_Utils_Array::extend($errors, self::sendReminderSms($tokenRow, $actionSchedule, $dao->contactID));
+              CRM_Utils_Array::extend($errors, self::sendReminderSms($tokenRow, $actionSchedule, $dao->contactID, $mapping, $dao->entity_id, $dao->reminderID, $dao->case_id));
             }
 
             if ($actionSchedule->mode == 'Email' or $actionSchedule->mode == 'User_Preference') {
-              CRM_Utils_Array::extend($errors, self::sendReminderEmail($tokenRow, $actionSchedule, $dao->contactID));
+              CRM_Utils_Array::extend($errors, self::sendReminderEmail($tokenRow, $actionSchedule, $dao->contactID, $mapping, $dao->entity_id, $dao->reminderID, $dao->case_id));
```

Comment:
@jitendrapurohit this is failing in the test suite on "Undefined property: CRM_Core_DAO::$case_id"